file_formats/ie_formats/ini.htm

```diff
@@ -1,89 +1,408 @@
+        <br />
+
+        <strong>script_combat = resref</strong><br />
+        This attribute defines the combat script assigned to the creature when it is spawned. Omit this attribute to use the script defined by the CRE resource. This script slot is only available in IWD2.<br />
+        <br />
+
+        <strong>script_special_3 = resref</strong><br />
+        This attribute defines the special 3 script assigned to the creature when it is spawned. Omit this attribute to use the script defined by the CRE resource. This script slot is only available in IWD2.<br />
+        <br />
+
+        <strong>script_movement = resref</strong><br />
+        This attribute defines the movement script assigned to the creature when it is spawned. Omit this attribute to use the script defined by the CRE resource. This script slot is only available in IWD2.<br />
+        <br />
+
+        <strong>dialog = resref</strong><br />
+        This attribute defines the dialog assigned to the creature when it is spawned. Omit this attribute to use the dialog defined by the CRE resource.<br />
+        <br />
+
+        <strong>good_mod = number</strong><br />
+        This attribute defines the value that is added to the GOOD variable when the spawned creature is killed. It is only supported by PST and PSTEE.<br />
+        <br />
+
+        <strong>law_mod = number</strong><br />
+        This attribute defines the value that is added to the LAW variable when the spawned creature is killed. It is only supported by PST and PSTEE.<br />
+        <br />
+
+        <strong>lady_mod = number</strong><br />
+        This attribute defines the value that is added to the LADY variable when the spawned creature is killed. It is only supported by PST and PSTEE.<br />
+        <br />
+
+        <strong>murder_mod = number</strong><br />
+        This attribute defines the value that is added to the MURDER variable when the spawned creature is killed. It is only supported by PST and PSTEE.<br />
+        <br />
+
+        <strong>death_scriptname = boolean_value</strong><br />
+        This attribute indicates whether the script name defined by the "script_name" attribute is also used for the death variable.<br />
+        <br />
+
+        <strong>death_faction = boolean_value</strong><br />
+        <span class="unknown">It is likely that this value controls whether the faction death counter is incremented when the spawned creature is killed.</span> It is only supported by PST and PSTEE.<br />
+        <br />
+
+        <strong>death_team = boolean_value</strong><br />
+        <span class="unknown">It is likely that this value controls whether the team death counter is incremented when the spawned creature is killed.</span> It is only supported by PST and PSTEE.<br />
+        <br />
```

Comment:
I consider these two confirmed. Avenger implemented them when he was reverse engineering and my little data foray showed there is a base for that.

Reply:
I can't verify that for PSTEE. Specifying these values (as well as `ai_faction` and `ai_team`) didn't apply any persistent changes.

Reply:
Interesting. We can at minimum mention that and the fact that _team and _faction are unused in the games.